samples/java-valueentity-counter-pojo/src/test/java/com/example/domain/CounterTest.java

```diff
@@ -0,0 +1,74 @@
+/* This code was generated by Akka Serverless tooling.
+ * As long as this file exists it will not be re-generated.
+ * You are free to make changes to this file.
+ */
+package com.example.domain;
+
+// tag::sample-unit-test[]
+import com.akkaserverless.javasdk.testkit.ValueEntityResult;
+import com.example.CounterApi;
+import com.google.protobuf.Empty;
+import org.junit.Test;
+import org.example.domain.CounterState;
+import static org.junit.Assert.*;
+
+public class CounterTest {
+
+  @Test
+  public void increaseCounterTest() {
+    CounterTestKit testKit = CounterTestKit.of(Counter::new);
+
+    CounterApi.IncreaseValue increaseValueCommand = CounterApi.IncreaseValue.newBuilder()
+        .setValue(1)
+        .build();
+    ValueEntityResult<Empty> result1 = testKit.increase(increaseValueCommand);
+    assertEquals(Empty.getDefaultInstance(), result1.getReply());
+    assertEquals(1, testKit.getState().value);
```

Comment:
The testkit just works out of the box? Neat.

Reply:
Yep! They are just java types.